extension/src/frequency.rs

```diff
@@ -568,6 +806,40 @@ requires = [topn_agg_trans, space_saving_final, space_saving_combine, space_savi
 
 extension_sql!("\n\
     CREATE AGGREGATE toolkit_experimental.topn_agg(\n\
+        integer, INT8\n\
+    ) (\n\
+        sfunc = toolkit_experimental.topn_agg_bigint_trans,\n\
+        stype = internal,\n\
+        finalfunc = toolkit_experimental.space_saving_bigint_final,\n\
+        combinefunc = toolkit_experimental.space_saving_combine,\n\
+        serialfunc = toolkit_experimental.space_saving_serialize,\n\
+        deserialfunc = toolkit_experimental.space_saving_deserialize,\n\
+        parallel = safe\n\
+    );\n\
+",
+name = "topn_bigint_agg",
+requires = [topn_agg_bigint_trans, space_saving_bigint_final, space_saving_combine, space_saving_serialize, space_saving_deserialize],
+);
+
+extension_sql!("\n\
+    CREATE AGGREGATE toolkit_experimental.topn_agg(\n\
+        integer, TEXT\n\
```

Comment:
the arguments to the aggregates should have names, one because I the update scripts will need them once stable, but more importantly for documentation in `\df`. Actually, can this use the aggregate builder instead?